packages/api/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ## [UNRELEASED]
 
+## [1.1.0](https://github.com/cennznet/api.js/compare/release/1.0.3...release/1.1.0) (31/07/2020)
+  Add doughnut + CENNZnut support and e2e tests
```

Comment:
- Now that adding sufficient tests is essential, do we need to address `e2e tests` in every release items?
- Shall we follow the practice of [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) to  address types of changes? It may roughly look like:

```
## [1.1.0] - 2020-07-31
### Added
- Add doughnut and CENNZnut support
```

Reply:
Ok..will take this into consideration for next time..

Reply:
Sure, may need to check with @alexsednz before getting it merged, due to conflicts with https://github.com/cennznet/api.js/pull/138

Reply:
This release branch should get merged before anything else